### PR TITLE
skip pattern-where-python if rce flag not enabled

### PR DESCRIPTION
**Pull request by brendongo**

### CODE REVIEW

File: semgrep/semgrep/evaluation.py [hunk 1]
```diff
@@ -66,10 +65,10 @@ def _evaluate_single_expression(
         return output_ranges
     elif expression.operator == OPERATORS.WHERE_PYTHON:
         if not flags or flags[RCE_RULE_FLAG] != True:
-            print_error_exit(
-                f"at least one rule needs to execute arbitrary code; this is dangerous! if you want to continue, enable the flag: {RCE_RULE_FLAG}",
-                NEED_ARBITRARY_CODE_EXEC_EXIT_CODE,
+            print_error(
```
Comment (ievans) [inline semgrep/semgrep/evaluation.py:68]: @brendongo if --strict is set, I think we should still exit with NEED_ARBITRARY_CODE_EXEC_EXIT_CODE

Comment (brendongo) [reply semgrep/semgrep/evaluation.py:68]: getting --strict down this deep into evaluation will be non-0 work so will probably move this to next release to focus on the other PRs.

Comment (DrewDennison) [inline semgrep/semgrep/evaluation.py:68]:
```suggestion
            print_error_exit(
```

**Comment by brendongo** [comment]
Will create a new PR to do this.

**Pull request closed without merging**